tests/integration/ceph_object_test.go

```diff
@@ -887,10 +887,11 @@ func testObjectStoreOperations(s *suite.Suite, helper *clients.TestClient, k8sh
 
 		t.Run("lifecycle was removed from bucket", func(t *testing.T) {
 			var err error
-			utils.Retry(20, time.Second, "lifecycle is gone", func() bool {
-				_, err = s3client.Client.GetBucketLifecycleConfiguration(&s3.GetBucketLifecycleConfigurationInput{
+			utils.Retry(600, time.Second, "lifecycle is gone", func() bool {
```

Comment:
600 seconds is a really long time, but 5 minutes also seems like a reasonable periodicity the RGW might use to async remove lifecycles. In the event the periodicity is as long as 5 minutes, this should verify whether it's a timing issue or an issue of RGW just not doing the right thing